src/detail-page/components/TabPanel.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface TabPanelProps {
+    children?: React.ReactNode;
+    index: unknown;
+    value: unknown;
```

Comment:
Why is it unnknown? It looks like it should be a number for index and value.

Reply:
Done!